packages/python-packages/apiview-copilot/main.tsp

```diff
@@ -0,0 +1,123 @@
+// main.tsp - TypeSpec API surface for apiview-copilot FastAPI app
+
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI3;
+
+@service({
+  title: "APIView Copilot Service",
+  version: "1.0.0"
+})
+@useDependency(RestResource)
+@useDependency(OpenAPI3Resource)
+
+namespace ApiViewCopilot {
+
+  @route("/{language}")
+  @post
+  @deprecated("Use /api-review/start instead")
```

Comment:
there is not `@deprecated` decorator, you could add a `@doc` to mention that this is deprecated

Reply:
https://typespec.io/docs/standard-library/reference/js-api/interfaces/deprecateddirective/#_top

Ah, I guess it's a directive.